web/frontend/src/components/slam.vue

```diff
@@ -95,6 +104,77 @@ const fetchSLAMPose = (name: string): Promise<commonApi.Pose> => {
   });
 };
 
+const executeMoveOnMap = async () => {
+  moveClick = false;
+
+  /*
+   * set request name
+   * here we set the name of the motion service the user is using
+   */
+  motionServiceReq.setName('builtin');
+
+  const value = await fetchSLAMPose(props.name);
+  // set pose in frame
+  const destination = new commonApi.Pose();
+  destination.setX(Math.abs(destinationMarker.x - value.getX()));
```

Comment:
This always sets X to a non-negative value, which seems unexpected.

Reply:
`destination.setX(destinationMarker.x)` is correct.
i used `destination.setX(Math.abs(destinationMarker.x - value.getX()))` previously because I was unsure if threeJS land coordinates were the same as value's.